ImprovedAnvil/TODO

```diff
@@ -1,3 +1,33 @@
+Bulian's Todo:
```

Comment:
I would appreciate if this was formatted in the same way as the original file and contained only things that aren't done yet